src/constants.js

```diff
@@ -9,6 +9,7 @@ const StageConstants = {
 	DEV_CONSOLE_TRANSPORTER_API_KEY: 'UDPWeb1',
 	AIO_CLI_API_KEY: 'aio-cli-console-auth-stage',
 	SMS_BASE_URL: 'https://graph-stage.adobe.io/api-admin',
+	EDGE_MESH_BASE_URL: 'https://edge-stage-graph.adobe.io/api',
```

Comment:
The value of `EDGE_MESH_SANDBOX_BASE_URL` in case `Stage` mode would be `undefined`. Is that acceptable for the CLI?

Reply:
The current implementation does not lead to a scenario the `EDGE_MESH_SANDBOX_BASE_URL` env variable is referenced for the Stage CLI. We can define such a constant with the same `edge-stage-graph` url anyway, but I prefer not to define a value that bears no relevance to the business logic.

That said, I don't feel strongly about it either way 🙂